a5/Dockerfile

```diff
@@ -0,0 +1,7 @@
+# rosindustrial/a5:kinetic
+FROM rosindustrial/noether:kinetic
+MAINTAINER Austin.Deric@gmail.com
```

Comment:
should we just get rid of the maintainer line or would you suggest we change it to `LABEL maintainer Austin.Deric@gmail.com`?